admin/jobs/river/validate_deployments.go

```diff
@@ -0,0 +1,147 @@
+package river
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/rilldata/rill/admin"
+	"github.com/rilldata/rill/admin/database"
+	"github.com/rilldata/rill/runtime/pkg/observability"
+	"github.com/riverqueue/river"
+	"go.uber.org/zap"
+)
+
+type ValidateDeploymentsArgs struct{}
+
+func (ValidateDeploymentsArgs) Kind() string { return "validate_deployments" }
+
+type ValidateDeploymentsWorker struct {
+	admin *admin.Service
+	river.WorkerDefaults[ValidateDeploymentsArgs]
+}
+
+func (w *ValidateDeploymentsWorker) Work(ctx context.Context, job *river.Job[ValidateDeploymentsArgs]) error {
+	return work(ctx, w.admin.Logger, job.Kind, w.validateDeployments)
+}
```

Comment:
It's a shame to need to wrap all work functions like this, but not sure I have a great alternative suggestion – seems like the client itself doesn't have any hooks for this. If the River logger is changed to info-level, does it handle this for us by any chance?

Or could we maybe have some kind of approach similar to otel spans? Something like:
```go
func (w *Worker) Work(ctx context.Context, ...) (err error) {
  ctx := w.client.start(ctx, job)
  defer func() { w.client.done(ctx, job, err) } // Can use e.g. trace.SpanFromContext(ctx) to get the span and close it
}
```

Not sure if that's any better though...